runner/runners/queue.go

```diff
@@ -109,14 +110,20 @@ type QueueController struct {
 	// used to signal a cmd run request
 	reqCh chan interface{}
 	// used to signal a request to update the Filer
-	updateCh <-chan time.Time
+	updateCh chan struct{}
 }
 
-func makeSimpleTicker(d time.Duration) <-chan time.Time {
-	if d != snapshot.NoDuration {
-		return time.NewTicker(d).C
+// Start a goroutine that forwards update ticks to updateCh, so that we can skip if e.g. init failed
+func startUpdateTicker(d time.Duration, u chan<- struct{}) {
+	if d == snapshot.NoDuration || u == nil {
+		return
 	}
-	return nil
+	ticker := time.NewTicker(d).C
```

Comment:
agreed, I left it out because there's no close/stop implementation for queue controller/runner in general, but will add small comment